### ISSUE TITLE
Make the plugin work with CheckMK 2.3.0 

### ISSUE THREAD

**Issue by 47k** [elcamlost/checkmk-php-fpm-plugin#16]
Can you please make the plugin work with CheckMK 2.3.0

**Comment by elcamlost** [comment]
Hi @47k @SjamonDaal I've recently release a new version (both here under Release and in checkmk exchange) which should 2.3 (but not 2.4) compatible. Feel free to give it a try.

I will keep this issue open for a few weeks, so you have a chance to test it and share the feedback.

**Comment by 47k** [comment]
Yes thank you - I saw this on the CheckMK Exchange and installed it already at friday. Seem fine - thank you!!! :) I waited for so long for this <3 

Only the Agent Bakery plugin installation is not working or not implemented?:

![Image](https://github.com/user-attachments/assets/8a189243-6709-41ef-ab81-876ddc2399df)

**Comment by elcamlost** [comment]
@47k agent bakery is not implemented. See #14 for the details.

**Comment by elcamlost** [comment]
I'm closing this one since it seems to work with 2.3.
@47k thanks for confirming it works and thanks for your patience. Hope, I would be able to make it compatible with 2.4 much sooner 😉
